Dockerfile

```diff
@@ -16,10 +16,9 @@ RUN apt update -y && \
         busybox \
         netcat-openbsd \
         procserv \
-        wget \
         $([ -n "$RUNTIME_PIP_PACKAGES" ] && echo pip) \
         $RUNTIME_PACKAGES && \
-    ln -s /bin/busybox /usr/bin/unzip && \
+    busybox --install && \
```

Comment:
Does busybox wget support HTTPS?

Reply:
On local tests I've done, it seems to support TLS, yes. [On their documentation page][1], there is nothing about this, though.

[1]: https://www.busybox.net/downloads/BusyBox.html

Reply:
Users should be aware that they need to add `ca-certificates` to `RUNTIME_PACKAGES`, though. Debian image currently does not add it by default; there is an issue open for that: https://github.com/debuerreotype/docker-debian-artifacts/issues/15.

Maybe that would be something to document, but I'm not sure if it pays off.